pandas/tests/extension/arrow/bool.py

```diff
@@ -67,7 +67,11 @@ def _from_sequence(cls, scalars, dtype=None, copy=False):
         return cls.from_scalars(scalars)
 
     def __getitem__(self, item):
```

Comment:
see my comment above, we should create an Indexing EA mixin and use the interface here